include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -8468,8 +8468,23 @@ FEFaceEvaluation<dim,
     {
       this->face_orientations[0] = 0;
       this->face_numbers[0]      = face_number;
-      for (unsigned int i = 0; i < n_lanes; ++i)
-        this->cell_ids[i] = cell_index * n_lanes + i;
+      if (this->matrix_free->n_active_entries_per_cell_batch(this->cell) ==
+          n_lanes)
+        {
+          DEAL_II_OPENMP_SIMD_PRAGMA
```

Comment:
Do we need this?

Reply:
I have copied it from here https://github.com/dealii/dealii/blob/c03f51671b0d04ede557bf9a0a1a893c470d5150/include/deal.II/matrix_free/fe_evaluation.h#L7488-L7502